master/pkg/schemas/expconf/experiment_config.go

```diff
@@ -43,6 +43,8 @@ type ExperimentConfigV0 struct {
 	RawSecurity                 *SecurityConfigV0           `json:"security,omitempty"`
 	RawTensorboardStorage       *TensorboardStorageConfigV0 `json:"tensorboard_storage,omitempty"`
 	RawWorkspace                *string                     `json:"workspace"`
+	RawSlurmClusterConfig       *SlurmClusterConfigV0       `json:"slurm"`
```

Comment:
maybe just `SlurmConfigV0`, `PbsConfigV0`? the 'cluster' is a bit weird.

Reply:
I'll change the names.